docs/index.html

```diff
@@ -288,20 +288,54 @@ <h2>#TwitterThrowback</h2>
     var submitButton = document.getElementById('submit-button');
     var loadingIndicator = document.getElementById('loading-indicator');
 </script>
+<script>
+    /**
+     * @function getMonthAndDay
+     * @description returns the Date API Friendly month and day 
+     * @param monthAndDayString a string containing the month and day separated by a space
+    */
+     function getMonthAndDay(monthAndDayString) {
+        const todayAsArray = [new Date().getMonth() + 1, new Date().getDate()];
```

Comment:
I have to say, I love the naming.

Reply:
reading `clean code` seems to be paying off